sdks/python/apache_beam/io/gcsio.py

```diff
@@ -21,13 +21,13 @@
 https://github.com/GoogleCloudPlatform/appengine-gcs-client.
 """
 
+import cStringIO as StringIO
```

Comment:
Mention why cStringIO is better with a reference to "https://docs.python.org/2/library/stringio.html" ?